game_eggs/gta/openmp/egg-open-m-p.json

```diff
@@ -0,0 +1,42 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-05-24T23:37:44+02:00",
+    "name": "OpenMP",
+    "author": "vedrancappone@gmail.com",
+    "description": "A multiplayer mod for Grand Theft Auto: San Andreas that is fully backwards compatible with San Andreas Multiplayer.",
+    "features": null,
+    "docker_images": {
+        "Samp": "ghcr.io\/parkervcp\/games:samp"
+    },
+    "file_denylist": [],
+    "startup": ".\/omp-server",
+    "config": {
+        "files": "{\r\n    \"config.json\": {\r\n        \"parser\": \"json\",\r\n        \"find\": {\r\n            \"network.port\": \"{{server.build.default.port}}\",\r\n            \"rcon.password\": \"{{server.build.env.RCON_PASSWORD}}\"\r\n        }\r\n    }\r\n}",
+        "startup": "{}",
+        "logs": "{}",
+        "stop": "^^C"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\r\n\r\necho \"running: curl -sSL -o openmp.tar.gz https:\/\/github.com\/openmultiplayer\/open.mp\/releases\/download\/v1-RC2\/open.mp-linux-x86-dynssl.tar.gz\"\r\ncurl -sSL -o openmp.tar.gz https:\/\/github.com\/openmultiplayer\/open.mp\/releases\/download\/v1-RC2\/open.mp-linux-x86-dynssl.tar.gz\r\n\r\nmkdir -p \/mnt\/server\r\ntar -xzvf openmp.tar.gz --strip-components=1 -C \/mnt\/server\/\r\n\r\ncd \/mnt\/server || exit\r\n\r\necho \"creating default config\"\r\n\r\nif [ -e conf.json ]; then\r\n    echo \"server config file exists\"\r\nelse\r\n    echo \"Downloading default OpenMp config\"\r\n    curl https:\/\/raw.githubusercontent.com\/parkervcp\/eggs\/master\/game_eggs\/gta\/openmp\/confing.json >> confing.json\r\nfi\r\n\r\nchown -R root:root \/mnt\r\n\r\nexport HOME=\/mnt\/server\r\n\r\n\r\necho \"-----------------------------------------\"\r\necho \"Installation completed...\"\r\necho \"-----------------------------------------\"",

```

Comment:
Please do not hardcode the sersion but use our github release grab script: https://github.com/parkervcp/eggs/blob/master/scripts/github_release_grabber.sh

This:
```bash
chown -R root:root /mnt

export HOME=/mnt/server
```
Is not needed as that will be auto done by the wings